### PR TITLE
Check while typing menu

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

File: electron/js/menu/system.js [hunk 1]
```diff
@@ -238,6 +238,17 @@ var editTemplate = {
...
     {i18n: 'menuSelectAll', role: 'selectall'},
+    separatorTemplate,
+    {
+      i18n: 'menuSpelling',
+      type: 'checkbox',
+      checked: init.restore('spelling', true) && config.SPELL_SUPPORTED.indexOf(locale.getCurrent()) > -1,
+      enabled: config.SPELL_SUPPORTED.indexOf(locale.getCurrent()) > -1,
+      click: function(event) {
+        init.save('spelling', event.checked);
+        console.log(event.checked);
```
Comment (gregor) [inline electron/js/menu/system.js]: Debug code

Comment (herrmannplatz) [inline electron/js/menu/system.js]: Boooooooo

**Comment by gregor** [comment]
I also added the German translations to Crowdin

**Comment by lipis** [comment]
There

**Pull request merged**